livre/eq/settings/CameraSettings.h

```diff
@@ -63,13 +64,33 @@ class CameraSettings : public co::Distributable< ::lexis::render::LookOut >
     void setCameraPosition( const Vector3f& position );
 
     /**
+     * Register the function to publish the camera to outer world
+     * @param position
+     */
+    void registerPublishFunction( std::function< void( const Matrix4f& )> publishFunc );
+
+    /**
      * Sets the camera orientation, defined by a lookAt vector.
      * @param lookAt the reference point position to orient the camera to.
      */
     void setCameraLookAt( const Vector3f& lookAt );
 
+    /**
+     * Sets the camera's modelview matrix.
+     * @param modelview the new modelview matrix.
+     */
+    void setModelViewMatrix( const Matrix4f& modelview );
+
     /** @return the camera transformation */
     Matrix4f getModelViewMatrix() const;
+
+private:
+
+    virtual void serialize( co::DataOStream& os, const uint64_t dirtyBits );
+    virtual void deserialize( co::DataIStream& is, const uint64_t dirtyBits );
+
+    Matrix4f _modelview;
+    std::function< void( const Matrix4f& )> _publishFunc;
```

Comment:
Why cameraModified ? This function is publishCamera(). _publishFunc seems more appropriate.